executor/show_test.go

```diff
@@ -944,6 +944,49 @@ func (s *testSuite5) TestShowCreateTable(c *C) {
 			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin"))
 }
 
+func (s *testAutoRandomSuite) TestShowCreateTablePlacement(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+
+	// case for direct opts
+	tk.MustExec(`DROP TABLE IF EXISTS t`)
+	tk.MustExec("create table t(a int) " +
+		"PRIMARY_REGION=\"cn-east-1\" " +
+		"REGIONS=\"cn-east-1, cn-east-2\" " +
+		"FOLLOWERS=2 " +
+		"FOLLOWER_CONSTRAINTS=\"[+zone=cn-east-1]\" " +
+		"CONSTRAINTS=\"[+disk=ssd]\"")
+	tk.MustQuery(`show create table t`).Check(testutil.RowsWithSep("|",
+		"t CREATE TABLE `t` (\n"+
+			"  `a` int(11) DEFAULT NULL\n"+
+			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin "+
+			"/*T![placement] PRIMARY_REGION=\"cn-east-1\" */ "+
+			"/*T![placement] REGIONS=\"cn-east-1, cn-east-2\" */ "+
+			"/*T![placement] FOLLOWERS=2 */ "+
+			"/*T![placement] CONSTRAINTS=\"[+disk=ssd]\" */ "+
+			"/*T![placement] FOLLOWER_CONSTRAINTS=\"[+zone=cn-east-1]\" */",
```

Comment:
I have a interesting idea to check whether you show result is right:

you can fetch the show result and use it to create table again. check diff of this two tables

Reply:
check `special_cmt_ctrl.go` for more feature comment control detail

Reply:
It seems not work. shoud we open a new issue to support DDL with `/*T!`?

Reply:
I think there are at least two more features about special comment `/*T[placement]*/`:
1. DDL syntax support parse `/*T[placement]*/`
2. binlog downstream DDL compatibility support, which depends on the first one.

Shall we split it?

Reply:
related to https://github.com/pingcap/parser/pull/1334

Reply:
thinks, that's good